package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Leader-Board-Setup",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
- [ ] Please make sure to include an `npm start` script, so you can run your app from the console.